src/data/markdown/translated-guides/en/02 Using k6/14 Scenarios/00 About scenarios.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "About scenarios"
+excerpt: High-level explanations about how your executor configuration can change the test execution and test results
+---
+
+These topics explain the essential concepts of how scenarios and their executors work.
+
+Different scenario configurations can affect many different aspects of your system,
+including the generated load, utilized resources, and emitted metrics.
+If you know a bit about how scenarios work, you'll both design better tests for resources and goals, and interpret test results with more understanding.
```

Comment:
I am not sure I understand "design better tests for resources and goals"

Reply:
The thinking was that if you understand scenarios better, you can:

- Make better decisions for your test goals, because certain scenarios correspond better to certain test design. To spike test a single component for raw throughput probably requires an arrival rate executor. To just see how quickly your system can churn through x number of iterations, `shared iterations` is a simpler choice.
- Use resources better, because understanding not to use maxVUs means you'll use CPU cycles more efficiently.

Of course now I realize that this is an enormous amount of implied information.

Will change to:
>If you know a bit about how scenarios work, you'll design better tests and interpret test results with more understanding.